testdata.csv

```diff
@@ -0,0 +1,9 @@
+24473694, 0120000000064218
+24050377, 0120000000064117
+35951196, 0120000000069271
+40244948, 0120000000073114
+27787623, 0120000000073821
+46822480, 0120000000077238
+79151115, 0120000000078268
+65416574, 0120000000079302
+95635593, 0120000000083747
```

Comment:
Is this used for anything?

Reply:
Nope! My bad, just a random file I generated. Shouldn't be in there. Will sort